src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -79,6 +80,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $name = $input->getArgument('name');
         $helper = new DescriptorHelper($this->fileLinkFormatter);
         $routes = $this->router->getRouteCollection();
+        $sortOption = $input->getOption('sort');
```

Comment:
Maybe `$sortProperty` fits better?

Reply:
I chose `sortOption` between `sortProperty` and `sortField` because I felt it was closer to getOption, but I can rename it if you feel it's more pertinent.

Reply:
BTW when you suggested using a `class` constant, did you refer to the mocked container?